contracts/script/deploy/verifiable/doc/DEPLOYMENT_PROCEDURE.md

```diff
@@ -0,0 +1,30 @@
+# Deployment Procedure
```

Comment:
nit could just rename file to something more succint like `DEPLOY.md`

Reply:
https://github.com/Layr-Labs/eigenda/pull/1408/commits/2f9eff21c624fa455b76c07370ee39f4c7f1e935